packages/evm/contracts/budgets/AManagedBudget.sol

```diff
@@ -32,6 +34,19 @@ abstract contract AManagedBudget is ABudget, IERC1155Receiver {
     /// @return The amount of assets distributed
     function distributed(address asset_, uint256 tokenId_) external view virtual returns (uint256);
 
+    // Optionally override the function if needed
+    function clawbackFromTarget(address target, bytes calldata data_, uint256 boostId, uint256 incentiveId)
+        external
+        virtual
+        override
+        onlyAuthorized
+        returns (uint256)
+    {
+        AIncentive.ClawbackPayload memory payload = AIncentive.ClawbackPayload({target: address(this), data: data_});
+
+        return IClaw(target).clawback(abi.encode(payload), boostId, incentiveId);
+    }
+
```

Comment:
might as well just make this a function interface or delete it if solidity will let you.